spec/playload/parser/helpers/create_ratings_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+RSpec.describe Parser::Helpers::CreateRatings do
+  let(:result)       { described_class.new(new_params).call(call_params) }
+  let(:data)         { File.read('spec/support/sample_data.json') }
+  let(:game_details) { JSON.parse(data)['GameInfo'][0] }
+
+  let(:release)            { build(:release) }
+  let(:rating)             { build(:rating) }
+  let(:rating_description) { build(:rating_description) }
+
+  let(:rating_repo)                     { instance_double RatingRepository }
+  let(:release_rating_repo)             { instance_double ReleaseRatingRepository }
+  let(:rating_description_repo)         { instance_double RatingDescriptionRepository }
+  let(:release_rating_description_repo) { instance_double ReleaseRatingDescriptionRepository }
+
+  let(:call_params) { Hash[game_details: game_details, release: release] }
+
+  let(:new_params) do
+    {
+      rating_repo: rating_repo,
+      release_rating_repo: release_rating_repo,
+      rating_description_repo: rating_description_repo,
+      release_rating_description_repo: release_rating_description_repo
+    }
+  end
+
+  describe '#call' do
+    before do
+      allow(rating_repo).to receive(:find_by).and_return(nil)
+      allow(rating_repo).to receive(:create).and_return(rating)
+      allow(release_rating_repo).to receive(:create)
+      allow(rating_description_repo).to receive(:find_by).and_return(nil)
+      allow(rating_description_repo).to receive(:create).and_return(rating_description)
+      allow(release_rating_description_repo).to receive(:create)
+      result
+    end
+
+    describe 'creating ratings' do
```

Comment:
Optional: The spec output may read a little more nicely if these were contextual blocks, e.g. 'when creating ratings it tries to find...'